data/json/monsters.json

```diff
@@ -2066,6 +2083,7 @@
     "armor_bash":12,
     "armor_cut":8,
     "vision_day":50,
+    "vision_night":5,
```

Comment:
Ouch, agreeing with @KA101. I thought it was agreed HNV should keep the player on top of things?